android-core/src/main/java/com/mparticle/internal/ConfigManager.java

```diff
@@ -1276,6 +1283,28 @@ public ConsentState getConsentState(long mpid) {
         return ConsentState.withConsentState(serializedConsent).build();
     }
 
+    public boolean isDirectUrlRoutingEnabled() {
+        return directUrlRouting;
+    }
+
+    /* When Direct URL Routing is true, we create a new set of baseUrls that include the silo in the urls. mParticle API keys are prefixed with the
+     silo and a hyphen (ex. "us1-", "us2-", "eu1-").  us1 was the first silo,and before other silos existed, there were no prefixes and all apiKeys
+     were us1. As such, if we split on a '-' and the resulting array length is 1, then it is an older APIkey that should route to us1.
+     When splitKey.length is greater than 1, then splitKey[0] will be us1, us2, eu1, au1, or st1, etc as new silos are added*/
+    public String getPodPrefix() {
```

Comment:
let's use this instead of the getPodPrefix in the test class.  this way we will be adding unit test coverage to this method

Reply:
or copy and paste this logic to the test if this is not possible